packages/lib/src/data/PlayerStats.ts

```diff
@@ -92,4 +92,19 @@ export default class PlayerStats {
 		}
 		return json;
 	}
+
+	merge(other: PlayerStats) {
+		this.joinCount += other.joinCount;
+		this.onlineTimeMs += other.onlineTimeMs;
+		if (!this.firstJoinAt || (other.firstJoinAt && other.firstJoinAt > this.firstJoinAt)) {
```

Comment:
Why would you take the higehr number for firstJoinAt? Aren't we interested in the lowest number, ie when the player first joined?

Reply:
I had a reason about it being "the most recently updated" but your reasoning is better; I'll update to take the the earlier time.